scripts/benchmark/expandedpostings.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname $(readlink -e -v ${BASH_SOURCE[0]}))"
+
+cd ${DIR}/../../
+
+RUN="expandedpostings"
+mkdir -p ${DIR}/bench_outs/${RUN}
+
+for TEST in BenchmarkBucketIndexReader_ExpandedPostings/binary_header BenchmarkBucketIndexReader_ExpandedPostings/index_json; do
+    echo "Running ${TEST} run ${RUN}"
+    go test -bench=${TEST} -run=^$ -benchmem -memprofile ${DIR}/bench_outs/${RUN}/memprofile${TEST//\//-}.out -timeout 2h -benchtime 60s ./pkg/store > ${DIR}/bench_outs/${RUN}/bench${TEST//\//-}.out
+done
+
+sed 's/BenchmarkBucketIndexReader_ExpandedPostings\/binary_header/BenchmarkBucketIndexReader_ExpandedPostings/g' ${DIR}/bench_outs/${RUN}/benchBenchmarkBucketIndexReader_ExpandedPostings-binary_header.out > ${DIR}/bench_outs/${RUN}/bench_new.out
```

Comment:
So this part is related to comparing index-header to index-cache.json. Since we move to index-header, we will remove index-cache.json support soon.

That's why I would rather suggest refactoring this into script that will run benchmark on master vs here - but that might involve bit git checkout mangling. Not sure if worth to check them in and maintain...

Reply:
It is related to thanos version indeed, check them in may be more easily for users.